pysmt/operators.py

```diff
@@ -55,7 +55,11 @@
 BV_ASHR,                                    # Arithmetic shift right (47)
 ARRAY_SELECT,                               # Array Select (48)
 ARRAY_STORE,                                # Array Store (49)
-ARRAY_VALUE                                 # Array Value (50)
+ARRAY_VALUE,                                # Array Value (50)
+
+DIV,                                        # Arithmetic Division (51)
```

Comment:
I tried the followig script (note that the exponentiation operator is ** not ^ ...)
```python
import z3

a = z3.Int("a")
b = z3.Int("b")

c = a / b
d = a * (b ** -1)

f1 = (c == 2)
f2 = (d == 2)

s = z3.Solver()
s.add(f1)
res = s.check()
print("Result for %s: %s" % (f1, res))
if str(res) == "sat":
    print "  Model: " + str(s.model())

print("")
s = z3.Solver()
s.add(f2)
res = s.check()
print("Result for %s: %s" % (f2, res))
if str(res) == "sat":
    print "  Model: " + str(s.model())
```

On my machine this produces:
```
$ python test.py
Result for a/b == 2: sat
  Model: [b = 0, a = 38]

Result for a*b**-1 == 2: unknown
```

So, both seems to be representable in z3, but the second one returns unknown. Moreover, there seems to be a bug, because the model returned by the division yields an invalid model (unless division by 0 is undefined, and therefore free). I think this needs a bit more investigation...